src/fuzz_introspector/analyses/function_call_analyser.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2022 Fuzz Introspector Authors
```

Comment:
Could you add some more comments throughout the code?

Reply:
done